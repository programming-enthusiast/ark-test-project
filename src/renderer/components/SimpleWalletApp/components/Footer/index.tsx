import React from "react";

interface FooterPropType {
	className?: string;
}
const Footer = ({ className }: FooterPropType) => (
	<>
		<div
			className={`${className} text-center w-full py-16px bg-footer-background main-font text-font-color1`}
		>
			2021 @ ARK.io | All rights reserved
		</div>
	</>
);

export default Footer;
