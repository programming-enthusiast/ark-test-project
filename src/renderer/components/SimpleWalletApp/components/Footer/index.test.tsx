import React from "react";
import renderer from "react-test-renderer";

import Footer from "./";

describe("Footer Component", () => {
	let renderedComponent: any;

	beforeAll(() => {
		renderedComponent = renderer.create(
			<Footer className="fixed bottom-0 left-0" />
		);
	});

	it("should render correctly", () => {
		expect(renderedComponent.toJSON()).toMatchSnapshot();
	});
});
