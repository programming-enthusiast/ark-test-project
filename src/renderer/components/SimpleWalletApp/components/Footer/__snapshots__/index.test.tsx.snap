// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer Component should render correctly 1`] = `
<div
  className="fixed bottom-0 left-0 text-center w-full py-16px bg-footer-background main-font text-font-color1"
>
  2021 @ ARK.io | All rights reserved
</div>
`;
